server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/SetMessagesMethodStepdefs.java

```diff
@@ -135,4 +138,18 @@ public void setFlags(String message, List<String> keywords) throws Throwable {
             .discardContent();
         mainStepdefs.awaitMethod.run();
     }
+
+    @When("^the user set flags (.*) via messageIdProbe of \"([^\"]*)\" on \"([^\"]*)\" of user \"([^\"]*)\"")
+    public void setDraftFlag(List<String> keywords, String message, String mailbox, String mailboxOwner) throws Exception {
```

Comment:
Also, **why are we dealing with JMAP level concerns (keywords) on a Manager level (Probe)?**